MediaBrowser.Model/Entities/UserPermissions.cs

```diff
@@ -0,0 +1,19 @@
+namespace MediaBrowser.Model.Entities;
+
+/// <summary>
+/// Class to hold data on user permissions for lists.
+/// </summary>
+/// <param name="userId">The user id.</param>
+/// <param name="canEdit">Edit permission.</param>
+public class UserPermissions(string userId, bool canEdit = false)
```

Comment:
I don't believe we want to introduce primary constructors unless it serves a real purpose